applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -425,7 +428,11 @@ def _create_linear_solver(self):
     def _create_builder_and_solver(self):
         linear_solver = self.get_linear_solver()
         if self.settings["block_builder"].GetBool():
-            builder_and_solver = KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver)
+            bs_params = KratosMultiphysics.Parameters("{}")
```

Comment:
this should be done in C++ within the strategy and builder and solver class, not here!

Reply:
OK